style.css

```diff
@@ -1,80 +1,284 @@
+/* COLORS:
+yellow: rgb(255, 240, 86);
+dark-black: rgb(25, 25, 25);
+dark-grey: rgb(157,157,157);
+light-grey: rgb(223, 226, 219);
+white: rgb(255, 255, 255);
+*/
+
 body {
-  background: url('./lokole-shot.jpg') no-repeat center center fixed;
-  background-size: cover;
-  background-color: #6e655c;
-  font-size: 24px;
-  line-height: 34px;
-  font-family: serif;
-  margin: 0;
+  overflow-y: hidden;
+  font-size: 16px;
+  text-align: center;
+  color: rgb(25, 25, 25);
+  background-color: rgb(223, 226, 219);
 }
 
-.page-container {
+#website-content {
+  display: none;
```

Comment:
This can be replaced by adding the bootstrap utility class `hidden` in the html. Same for `modal-1-content` below.